bindings/nodejs/lib/Account.ts

```diff
@@ -413,7 +414,7 @@ export class Account {
 
     async getParticipationEvent(
         eventId: ParticipationEventId,
-    ): Promise<[ParticipationEvent, Node[]]> {
+    ): Promise<[ParticipationEventWithNodes]> {
```

Comment:
What do these brackets mean ? Is that an array ? Why ?

Reply:
The brackets would indicate that it's a tuple, but since there is a single value returned, I think this is wrong.